### ISSUE TITLE
Add precommit script to lint the README file

### ISSUE THREAD

**Issue by rogerluan** [Blackjacx/WWDC#60]
Lint the file with things such as:

- Removing trailing whitespace
- Auto-executing the progress.sh file

Caveat: precommit hooks don't work if people are modifying the README via GitHub's website 😞 

Open to discuss

**Comment by Blackjacx** [comment]
I like it, it's okay when the progress is not always correct. Every new submission via local git will fix it 👍 Also my little refactoring will fix it https://github.com/Blackjacx/WWDC/issues/57#issuecomment-651408238. With this we won't need point 2 at all.

**Comment by rogerluan** [comment]
You're right! I think it's more clever to do this in the CI, and leave precommit only for linting 🙏
